src/LivewireServiceProvider.php

```diff
@@ -32,6 +32,8 @@ public function register()
         ));
 
         $this->allowCertainExceptionsToBypassTheBladeViewHandler();
+
+        $this->mergeConfigFrom(__DIR__.'/../config/livewire.php', 'livewire');
```

Comment:
Do we still need this if we use the second param defaults?

Reply:
Nope.